templates/parcel_new.html

```diff
@@ -27,6 +27,21 @@
         window.location.href = $(this).val();
       });
 
+      $('#lot').on('change', function() {
+        var idLot = $(this).val()
+        var delivery_type = $('#delivery').val()
+        $.getJSON('/pick_products', {
```

Comment:
Generate the URL using the `url` template tag and store it in an attribute on `#lot`.
`<input id="lot" data-href="{% url 'pick_products' %}/>"`